src/mono/wasm/debugger/BrowserDebugProxy/MemberObjectsExplorer.cs

```diff
@@ -692,7 +692,7 @@ public static GetMembersResult FromValues(JArray values, bool splitMembersByAcce
                     result.Split(member);
                 return result;
             }
-            result.Result.AddRange(values);
+            result.Result.AddRange(values, removeUnderscoreProperties: true);
```

Comment:
Can we remove these at the end, when we are ready with the final result?

Reply:
We can but then we will have to iterate through the collection twice.

Reply:
I will make the removal optional in `FromValues` too. So that if someone wants to use `FromValues` one day but not to get a clean, ready-to-send result they can skip the removal. So far in all the places we are calling `FromValues` removal is fine.